tests/platform_tests/test_power_off_reboot.py

```diff
@@ -40,25 +39,35 @@ def teardown_module(duthosts, enum_supervisor_dut_hostname, conn_graph_facts, xc
     check_interfaces_and_services(duthost, interfaces, xcvr_skip_list, INTERFACE_WAIT_TIME)
 
 
-def _power_off_reboot_helper(kwargs):
+def _power_off_reboot_helper(kwargs, power_on_event=None):
     """
     @summary: used to parametrized test cases on power_off_delay
     @param kwargs: the delay time between turning off and on the PSU
     """
     pdu_ctrl = kwargs["pdu_ctrl"]
     all_outlets = kwargs["all_outlets"]
     power_on_seq = kwargs["power_on_seq"]
-    delay_time = kwargs["delay_time"]
-
     for outlet in all_outlets:
         logging.debug("turning off {}".format(outlet))
         pdu_ctrl.turn_off_outlet(outlet)
-    time.sleep(delay_time)
+
```

Comment:
@xincunli-sonic consider a case where PDU has a bug due to which the device was not powered off, in that case, wait_for_shutdown() should fail

Reply:
If PDU was not powered off, the wait_for_shutdown will check SSH state is absent as below code snippet. When check failed, it will raise exception to indicate power off assertion failed.

```python
res = localhost.wait_for(host=dut_ip,
                             port=SONIC_SSH_PORT,
                             state='absent',
                             search_regex=SONIC_SSH_REGEX,
                             delay=delay,
                             timeout=timeout,
                             module_ignore_errors=True)
```